docs/guides/development/graphql/graphql-apollo-an-introduction/index.md

```diff
@@ -0,0 +1,230 @@
+---
+slug: graphql-apollo-an-introduction
+author:
+  name: Cameron Laird
+  email: claird@phaseit.net
+description: 'Two to three sentences describing your guide.'
+og_description: 'Two to three sentences describing your guide when shared on social media.'
+keywords: ['what is apollo graphql', 'apollo graphql client', 'apollo graphql server']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2021-07-19
+modified_by:
+  name: Linode
+title: "Graphql Apollo: An Introduction"
+h1_title: "Graphql Apollo: An Introduction with Examples"
+enable_h1: true
+contributor:
+  name: Cameron Laird
+---
+
+Do you retrieve data from multiple on-line sources to such limited-bandwidth [edge devices](https://www.intel.com/content/www/us/en/edge-computing/edge-devices.html) as smartwatches, mobile telephone handsets, and [Internet-of-things](https://www.linode.com/docs/guides/development/iot/) (IoT) machines?  If so, you want to learn [GraphQL](https://graphql.org/), a maturing standard *query language* for accessing on-line data that particularly fits such applications.  GraphQL began as an internal project at Facebook in 2012.  After a public release in 2015, GraphQL was provided a permanent home with the [GraphQL Foundation](https://graphql.org/foundation/) in 2019. The GraphQL Foundation is under the umbrella of the open-source-oriented [Linux Foundation](https://linuxfoundation.org/).
```

Comment:
2018:  please change `... in 2019. The GraphQL Foundation ...` to `... in 2018. The GraphQL Foundation ...`

Reply:
I actually prefer, `After its public release in 2015, the [GraphQL Foundation](https://graphql.org/foundation/) was created in 2018 to provide a permanent home for GraphQL.`

Reply:
WOW, is this making me feel rusty:  I'm accustomed to seeing source code and documentation through Git.  I need a little practice viewing **content** this way.